nilearn/group_sparse_covariance.py

```diff
@@ -545,7 +545,7 @@ def fit(self, subjects, y=None):
             _group_sparse_covariance, memory_level=1)(
                 self.covariances_, n_samples, self.alpha,
                 tol=self.tol, max_iter=self.max_iter,
-                verbose=self.verbose - 1, debug=False)
+                verbose=max(0, self.verbose - 1), debug=False)
```

Comment:
> Then I suggest that we stop reviewing this PR till #366 is merged.

That's probably the best way to do it while saving time and hassle for everybody involved. I don't think #366 is that far away from being merged anyway.